backend/src/openarchiefbeheer/logging/templates/logging/destruction_list_aborted.txt

```diff
@@ -1,3 +1,11 @@
-{% load i18n %}{% blocktranslate trimmed with list_name=log.content_object.name groups=log.extra_data.user_groups|join:", " record_manager=log.user comment=log.extra_data.comment count counter=log.extra_data.user_groups|length %}
+{% load i18n %}{% spaceless %}
```

Comment:
Does the spaceless do anything? I thought that it just removed white space between HTML tags :thinking:

Reply:
It seems to strip the leading spaces making tests less annoying

Reply:
Nice!